{"name": "登录测试", "status": "passed", "attachments": [{"name": "log", "source": "a09026be-b1fd-4dcb-bc32-6fbc38041bd1-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "caseinfo", "value": "{'name': '用户不存在', 'request': {'methond': 'post', 'url': 'http://10.0.10.131/login?username=aaa&password=1qaz!QA', 'data': {'password': '1qaz!QA', 'username': 'aaa'}}, 'validate': {'code': -1, 'msg': '认证失败'}}"}], "start": 1639624911351, "stop": 1639624911510, "uuid": "a2e19053-2f1e-4dfe-94a0-2732bf2c8db3", "historyId": "4997552f3407f882f4611f7c524a4b9e", "testCaseId": "82e5b0a8e2b6f9add96c0789ee7552f3", "fullName": "interface_test.test_login.TestLongi#test_login", "labels": [{"name": "parentSuite", "value": "interface_test"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLongi"}, {"name": "host", "value": "XA342458"}, {"name": "thread", "value": "13472-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "interface_test.test_login"}]}